### PR TITLE
created sign up page

### PR DESCRIPTION
**Pull request by Rockyraven**
sign up page created

### CODE REVIEW

File: src/pages/signup-page/SignUpPage.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        <h1 className="login-tilte">Sign Up</h1>
+        <input
+          type="text"
+          id="outlined-password-input"
+          className="input-box"
+          label="user"
+          autoComplete="current-password"
+          placeholder="First Name"
+          name="firstName"
+          // value={firstName}
```
Comment (Shubhamkr2610) [inline src/pages/signup-page/SignUpPage.js:44]: remove unused code

File: src/pages/signup-page/SignUpPage.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        <input
+          type="text"
+          id="outlined-password-input"
+          className="input-box"
+          label="user"
+          autoComplete="current-password"
+          placeholder="Last Name"
+          name="lastName"
+          // value={lastName}
+          onInput={(e) => setFormData(prev => ({...prev, [e.target.name]: e.target.value}))}
```
Comment (Shubhamkr2610) [inline src/pages/signup-page/SignUpPage.js:56]: remove commented code

**Review by Shubhamkr2610 — APPROVED** [review]
good to go after removing the commented code

**Review by vishalg8454 — APPROVED** [review]
Feature work as expected but add validations to the sign-up form.

**Pull request merged**